_layouts/home.html

```diff
@@ -264,6 +264,18 @@ <h3>Legend</h3>
      * @returns {string}
      */
     function popupTemplate(shelter) {
+
+        // Takes care of null here
+        let shelterPets = shelter.pets_notes;
+        if (shelterPets === 'null' || shelterPets === null) {
+            shelterPets = 'Unknown';
+        }
+
+        let shelterNotes = shelter.notes
+        if (shelterNotes === 'null' || shelterNotes === null) {
+            shelterNotes = 'None';
```

Comment:
I think for a free-form text field like notes, an empty string `''` would be a better default than something like `'None'`. Granted, this is a future-thinking thing, not a current state thing, but I think it's easy to prevent future problems in this case. I'm thinking of possible downstream analytics or export use cases where the extra text just explodes the size of the export for no reason.

Reply:
As far as users go, a display of `'None'` seems more clear than an empty space. I'm not sure what you can customize about analytics. Exporting shelter data seems more like another API. I'm not sure we should sacrifice user clarity for convenience of data.

Reply:
Sorry @plocket -- I've edited my comment. You're right, I was thinking API.

Reply:
Can this thread be resolved?

Reply:
Not yet -- I think we still need to look at the conditional that's not doing what it's supposed to.